migrations.json

```diff
@@ -2,11 +2,51 @@
     "migrations": [
         {
             "cli": "nx",
-            "version": "16.5.0-beta.2",
-            "description": "Add test-setup.ts to ignored files in production input",
-            "implementation": "./src/migrations/update-16-5-0/add-test-setup-to-inputs-ignore",
-            "package": "@nx/jest",
-            "name": "add-test-setup-to-inputs-ignore"
+            "version": "16.6.0-beta.6",
+            "description": "Prefix outputs with {workspaceRoot}/{projectRoot} if needed",
+            "implementation": "./src/migrations/update-15-0-0/prefix-outputs",
+            "package": "nx",
+            "name": "16.6.0-prefix-outputs"
+        },
+        {
+            "cli": "nx",
+            "version": "16.6.0-beta.0",
+            "description": "Explicitly set 'updateBuildableProjectDepsInPackageJson' to 'true' in targets that rely on that value as the default.",
+            "factory": "./src/migrations/update-16-6-0/explicitly-set-projects-to-update-buildable-deps",
+            "package": "@nx/js",
+            "name": "explicitly-set-projects-to-update-buildable-deps"
+        },
+        {
+            "cli": "nx",
+            "version": "16.6.0-beta.0",
+            "description": "Add dependsOn like ensure-symlink or sync-deps to targets",
+            "implementation": "./src/migrations/update-16-6-0/add-depends-on",
+            "package": "@nx/react-native",
+            "name": "update-16-6-0-add-dependsOn"
+        },
+        {
+            "cli": "nx",
+            "version": "16.6.0-beta.0",
+            "description": "Update metro.config.js to use the new metro config format",
```

Comment:
@ajwootto looks like its from here.

Reply:
yeah but it seems like now theres two config files or something. Do we need both? Is the migration doing something that keeps things "working" but expects you to do some manual work to migrate to the new config?

Reply:
looked into this and basically what the migration is doing is writing a backup of your old metro config while replacing the main one with a newly generated one that doesn't have any of the custom configuration from the previous one ported over. The idea is that you're supposed to manually bring over anything custom from your old config. I did that and deleted the v71 version because it's not actually used